vendor/deps/cli-ui/lib/cli/ui/glyph.rb

```diff
@@ -24,28 +24,38 @@ def message
       # * +handle+ - The handle in the +MAP+ constant
       # * +codepoint+ - The codepoint used to create the glyph (e.g. +0x2717+ for a ballot X)
       # * +color+ - What color to output the glyph. Check +CLI::UI::Color+ for options.
+      # * +plain+ - A fallback plain string to be used in case glyphs are disabled
       #
-      def initialize(handle, codepoint, color)
+      def initialize(handle, codepoint, color, plain)
         @handle    = handle
         @codepoint = codepoint
         @color     = color
+        @plain     = plain
         @char      = Array(codepoint).pack('U*')
         @to_s      = color.code + char + Color::RESET.code
         @fmt       = "{{#{color.name}:#{char}}}"
 
         MAP[handle] = self
       end
 
+      # Fetches the actual character(s) to be displayed for a glyph
+      #
+      # ==== Returns
+      # Returns the glyph string
+      def char
+        CLI::UI::OS.current.supports_emoji? ? @char : @plain
+      end
+
       # Mapping of glyphs to terminal output
       MAP = {}
-      STAR      = new('*', 0x2b51,           Color::YELLOW) # YELLOW SMALL STAR (⭑)
-      INFO      = new('i', 0x1d4be,          Color::BLUE)   # BLUE MATHEMATICAL SCRIPT SMALL i (𝒾)
-      QUESTION  = new('?', 0x003f,           Color::BLUE)   # BLUE QUESTION MARK (?)
-      CHECK     = new('v', 0x2713,           Color::GREEN)  # GREEN CHECK MARK (✓)
-      X         = new('x', 0x2717,           Color::RED)    # RED BALLOT X (✗)
-      BUG       = new('b', 0x1f41b,          Color::WHITE)  # Bug emoji (🐛)
-      CHEVRON   = new('>', 0xbb,             Color::YELLOW) # RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK (»)
-      HOURGLASS = new('H', [0x231b, 0xfe0e], Color::BLUE)   # HOURGLASS + VARIATION SELECTOR 15 (⌛︎)
+      STAR      = new('*', 0x2b51,           Color::YELLOW, '*')   # YELLOW SMALL STAR (⭑)
+      INFO      = new('i', 0x1d4be,          Color::BLUE,   'i')   # BLUE MATHEMATICAL SCRIPT SMALL i (𝒾)
+      QUESTION  = new('?', 0x003f,           Color::BLUE,   '?')   # BLUE QUESTION MARK (?)
+      CHECK     = new('v', 0x2713,           Color::GREEN,  '√')   # GREEN CHECK MARK (✓)
+      X         = new('x', 0x2717,           Color::RED,    'X')   # RED BALLOT X (✗)
+      BUG       = new('b', 0x1f41b,          Color::WHITE,  'bug') # Bug emoji (🐛)
```

Comment:
Shouldn't the fallback for this be a single character?  Why not use 'b'?

Reply:
I felt like `b` wouldn't be very meaningful - I can imagine some app saying 'If you run into a b please contact XYZ'. Code-wise, it doesn't necessarily need to be a single character.

Reply:
How about `!` instead? I think that aligns more with the idea of something to watch out for. Do we actually use `BUG` anywhere in the CLI though?

Reply:
We don't, but this change is going into cli-ui, so it needs to be generic enough that it makes sense in any scenario.

While I think `!` is a good idea, I'm not sure if this emoji isn't used mid-sentence somewhere - although that logic could be applied to all of these and it would likewise not make too much sense.

Reply:
Yeah personally I think it's safe to assume these can't/wouldn't/shouldn't be used as stand-alone words.

Reply:
I agree, changing it to `!` in the upstream PR. Thanks!